cmd/peggo/orchestrator.go

```diff
@@ -263,6 +279,9 @@ func getOrchestratorCmd() *cobra.Command {
 	cmd.Flags().Bool(flagRelayBatches, false, "Relay transaction batches to Ethereum")
 	cmd.Flags().Int64(flagEthBlocksPerLoop, 2000, "Number of Ethereum blocks to process per orchestrator loop")
 	cmd.Flags().String(flagCoinGeckoAPI, "https://api.coingecko.com/api/v3", "Specify the coingecko API endpoint")
+	cmd.Flags().StringSlice(flagOracleProviders, []string{umeedpfconfig.ProviderBinance, umeedpfconfig.ProviderHuobi},
```

Comment:
Can we have the default be empty? What would that mean if they're empty?

Reply:
Well, it could be empty... but then no batch is going to be profitable which is not good for the relayer and he wouldn't be rewarded at all

Do you guys think we should force the numbers of providers? @alexanderbez @facundomedica